### PR TITLE
Maya: add support for Maya 2025

### PR DESCRIPTION
**Pull request by antirotor**
## Changelog Description
Adding support to Maya 2025

## Additional info
This is adding default values to settings, moving around `capture.py` to Maya addon (where it should be anyway) and fixing some Qt imports there.

> [!NOTE]
> Touching `pyproject.toml` is done just to calm codespell in pre-commit hook because of the TVPaint path.

## Testing notes:
- Run Maya 2025 and it should work
- Run Maya 2024 should work too
- Playblasting should work also

### CODE REVIEW

**Review by BigRoy — APPROVED** [review]
Nice. Looks good to me.

Didn't do a test run.

**Review by m-u-r-p-h-y — CHANGES_REQUESTED** [review]
the menu shows up correctly all tools seems to run ok just:

some playblasting issue

```
Traceback (most recent call last):
  File "C:\Users\murph\AppData\Local\Ynput\AYON\dependency_packages\ayon_2402141620_windows.zip\dependencies\pyblish\plugin.py", line 527, in __explicit_process
    runner(*args)
  File "c:\Users\murph\Documents\Ayon\REPO\ayon-core\client\ayon_core\hosts\maya\plugins\publish\extract_playblast.py", line 60, in process
    lib.render_capture_preset(preset)
  File "c:\Users\murph\Documents\Ayon\REPO\ayon-core\client\ayon_core\hosts\maya\api\lib.py", line 239, in render_capture_preset
    path = capture.capture(log=self.log, **preset)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "c:\Users\murph\Documents\Ayon\REPO\ayon-core\client\ayon_core\hosts\maya\vendor\python\capture.py", line 169, in capture
    with _independent_panel(width=width + padding,
  File "C:\Program Files\Autodesk\Maya2025\Python\Lib\contextlib.py", line 137, in __enter__
    return next(self.gen)
           ^^^^^^^^^^^^^^
  File "c:\Users\murph\Documents\Ayon\REPO\ayon-core\client\ayon_core\hosts\maya\vendor\python\capture.py", line 614, in _independent_panel
    screen_width, screen_height = _get_screen_size()
                                  ^^^^^^^^^^^^^^^^^^
  File "c:\Users\murph\Documents\Ayon\REPO\ayon-core\client\ayon_core\hosts\maya\vendor\python\capture.py", line 872, in _get_screen_size
    rect = QtWidgets.QDesktopWidget().screenGeometry(-1)
           ^^^^^^^^^^^^^^^^^^^^^^^^
AttributeError: module 'qtpy.QtWidgets' has no attribute 'QDesktopWidget'
```

**Comment by BigRoy** [comment]
> QDesktopWidget

[It has been removed in `Qt6`](https://doc.qt.io/qt-6/widgets-changes-qt6.html#qdesktopwidget-and-qapplication-desktop) and thus `capture.py` would need to be updated.

You will need to use `QScreen`. I believe we're doing that elsewhere already. For example getting the screen geometry [here](https://github.com/ynput/ayon-core/blob/a4f2f76fbf98becd22b75012abf1abfc09bbd69d/client/ayon_core/tools/publisher/widgets/screenshot_widget.py#L172-L173)

File: pyproject.toml [hunk 1]
```diff
@@ -92,7 +92,7 @@ line-ending = "auto"
 
 [tool.codespell]
 # Ignore words that are not in the dictionary.
-ignore-words-list = "ayon,ynput"
+ignore-words-list = "ayon,ynput,developpement"
```
[the remark this answers is not in the document]
Comment (antirotor) [reply pyproject.toml:95]: Sure, but codespell pre-commit hook will not allow to commit unless this is added to exceptions.

**Review by m-u-r-p-h-y — APPROVED** [review]
Maya 2025 publishing, including playblast creation works as expected

image

**Pull request merged**